### PR TITLE
Add support for Tinkerboard 2

### PR DESCRIPTION
**Pull request by p1r473**
Hi Linus, been a while. Hope all is well.
I have recently purchased a Tinkerboard 2 and am adding support for it.
I'd like you to check over the changes and make sure its implemented how youd like. If not, I am happy to make any changes.

### CODE REVIEW

**Review by linusg — COMMENTED** [review]
Looks great! Let's rename `TINKER_BOARD2` to `TINKER_BOARD_2` and `tinker-board2` to `tinker-board-2`, and give the README sections that mention Tinker Board an update, e.g. by adding a sentence below the Tinker Board code example: "For Tinker Board 2, use `BoardType.TINKER_BOARD`."

**Comment by p1r473** [comment]
Thanks Linus, I have made the changes. Let me know what you think.

File: README.md [hunk 1]
```diff
@@ -83,14 +83,26 @@ False
...
+### To use with ASUS Tinker Board 2:
+In `def __init__()` in file `__init__.py`:
+```
+board_type: BoardType = BoardType.TINKER_BOARD_2
+```
+In `def _create_argument_parser()` in file `cli.py`:
 ```
+default="tinker-board-2"
+```
+
```
Comment (linusg) [inline README.md]: Hmm, that's not really what I meant (and doesn't seem like an improvement over the current example. More like this:

````
```python
>>> from rpi_backlight import Backlight, BoardType
>>>
>>> backlight = Backlight(board_type=BoardType.TINKER_BOARD)
>>> # continue like above
```

When using the ASUS Tinker Board 2, set `board_type` to `BoardType.TINKER_BOARD_2`.
````

Comment (p1r473) [reply README.md]: How do we prompt the user to also make the changes in the second file too, cli?
As there was a change needed in both files for it to work.
I tried changing it in only one place, and it was still thinking I was using a Raspberry Pi.

Comment (p1r473) [reply README.md]: OK I made the changes as requested.
In my fork, once you accept the pull, I'll change the default in both places
Is there any way to have it so the default only needs to be set in one place?

Comment (p1r473) [reply README.md]: OK I made the changes as requested.

**Comment by linusg** [comment]
🚀

**Comment by p1r473** [comment]
Thanks Linus!

**Comment by linusg** [comment]
I'll release this in a new version later today, will let you know. :)

**Comment by p1r473** [comment]
By the way,
Is there any modifications we can make to initiate the board type as either a Pi, Tinker 1 or 2, and not have to initiate it in both CLI and also init files?

**Comment by linusg** [comment]
Done! https://pypi.org/project/rpi-backlight/2.3.0

>  Is there any modifications we can make to initiate the board type as either a Pi, Tinker 1 or 2, and not have to initiate it in both CLI and also init files?

I suppose if there's a reliable way to detect the board type, we could have automatic detection if no board type is explicitly provided - perhaps by checking the existence of each path in `_BACKLIGHT_SYSFS_PATHS`?

**Comment by p1r473** [comment]
> Done! https://pypi.org/project/rpi-backlight/2.3.0
> 
> > Is there any modifications we can make to initiate the board type as either a Pi, Tinker 1 or 2, and not have to initiate it in both CLI and also init files?
> 
> I suppose if there's a reliable way to detect the board type, we could have automatic detection if no board type is explicitly provided - perhaps by checking the existence of each path in `_BACKLIGHT_SYSFS_PATHS`?

I was thinking of using `/proc/device-tree/model`
```
root@Harbormaster:~# cat /proc/device-tree/model
ASUS Tinker Board 2 (Linux Opensource)
```
What is the output on the Pi?

**Pull request merged**